boards/shields/nrf7002ek/Kconfig.shield

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config SHIELD_NRF7002EK
+	def_bool $(shields_list_contains,nrf7002ek)
```

Comment:
```
config SHIELD_NRF7002EK_NRF7000
	def_bool $(shields_list_contains,nrf7002ek_nrf7000)


config SHIELD_NRF7002EK_NRF7001
	def_bool $(shields_list_contains,nrf7002ek_nrf7001)
```

IRRC it works without Kconfig.shield if CONFIG_SHIELD_NRF7002EK_* are not used anywhere, e.g. no Kconfig.defconfig.

Reply:
No, it doesn't recognize the shield without the `Kconfig.shield`, this is mandatory, see https://github.com/zephyrproject-rtos/zephyr/blob/main/cmake/modules/shields.cmake#L53. I will add other two.